.travis.yml

```diff
@@ -9,7 +9,7 @@ before_script:
 scala:
   - 2.11.11
 jdk:
-  - oraclejdk8
+  - openjdk11
```

Comment:
I think you could use `opendjk8` if you prefer to use the same version

Reply:
@JesusMtnez Thanks. I'm wary of creating more issues. We should consider updating the Scala/JDK versions but that would imply reviewing all exercises and check nothing is breaking.